setup.py

```diff
@@ -5,23 +5,8 @@
 from setuptools import setup, find_packages
 
 
-def get_requirements():
```

Comment:
any specific reasons for removing this code?

Reply:
@dgpatelgit ^^

Reply:
As there is no specific dependencies for this component, we don't need function. There is no requirements.in file for this component. So cleaned up the code.